CakePHP/ruleset.xml

```diff
@@ -96,6 +96,8 @@
 
     <!-- Slevomat sniffs-->
     <rule ref="SlevomatCodingStandard.Arrays.TrailingArrayComma"/>
+    <rule ref="SlevomatCodingStandard.Functions.RequireTrailingCommaInDeclaration"/>
+    <rule ref="SlevomatCodingStandard.Functions.RequireTrailingCommaInClosureUse"/>
```

Comment:
This one is probably ok. I don't think multi-line `use` statements occur that often.

Reply:
Is it multi-line only?

Reply:
Those are only in affect for multi-line, yes.

Reply:
I have no objection to multi-line sniffs. I don't think there's any real benefit for diffs since function declarations don't change though, but whatever. The sniff should have "multiline" in the name!

Reply:
I think it is implied - as trailing one on single line doesnt make any sense really.